alembic.ini.mako

```diff
@@ -21,6 +21,7 @@ script_location = CONST_alembic/main
 # sourceless = false
 
 sqlalchemy.url = ${sqlalchemy["url"]}
+version_table = c2cgeoportal_version
```

Comment:
No it's not needed and it can be dangers if we don't rename the table in the database...